### ISSUE TITLE
Code-Fehler bei Thema 06, Folie 19

### ISSUE THREAD

**Issue by xehpuk** [LukasKalbertodt/programmieren-in-rust#12]
Kasten rechts unten:
```rust
let b = Foo;
let c = &mut b;
```
> error[[E0596]](https://doc.rust-lang.org/stable/error-index.html#E0596): cannot borrow `b` as mutable, as it is not declared as mutable

Korrektur:
```rust
let mut b = Foo;
let c = &mut b;
```

**Comment by LukasKalbertodt** [comment]
Stimmt, danke für den Hinweis. Allerdings werde ich die Folien leider nicht mehr aktualisieren. Wenn ich die Quell-PPTX Datein mit dem aktuellen Powerpoint öffne, sind leider ein paar Dinge kaputt (z.B. Zeilen, die umbrechen). Eventuell wurde auch Fira Sans oder Fira Mono aktualisiert, sodass irgendwas kaputt geht. Wie dem auch sei: die Folien zu aktualisieren ist leider zu viel Aufwand gerade. Insbesondere ist dieser Fehler nicht so tragisch würde ich sagen. 

Danke trotzdem! Ich lasse das Issue auf, falls das anderen Menschen auch auffällt.
